test/components/details_menu_component_test.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class PrimerDetailsMenuComponentTest < Minitest::Test
```

Comment:
may we also add a test to check if the content is rendered?
I'd also like to add a test checking that the HTML tag can't be modified